golem_messages/message/__init__.py

```diff
@@ -48,114 +59,4 @@
 from golem_messages.message.concents import ForceReportComputedTask  # noqa
```

Comment:
we might consider removing the concent-related messages from the list as they are probably imported using more specific imports...